static/templates/stream_creation_form.hbs

```diff
@@ -1,6 +1,6 @@
 <div class="hide" id="stream-creation" tabindex="-1" role="dialog"
   aria-label="{{t 'Stream creation' }}">
-    <form id="stream_creation_form" class="form-inline">
+    <form id="stream_creation_form">
```

Comment:
What does `form-inline` do?  It appears to be a bootstrap thing.

Reply:
It's a bootstrap class. It's most common use is in setting display properties to various input elements in the form so that if there is sufficient space horizontally input elements are shown like:
`label: input element`.
But as we override almost all of the styles set by its class (checked using developer tools), there is no effect on design by removing it.
I'll move this to a separate PR along with other places in stream-edit page where I have checked it is can be safely removed.